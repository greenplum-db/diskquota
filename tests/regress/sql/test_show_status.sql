select * from diskquota.status();

\! gpconfig -c "diskquota.hard_limit" -v "on" > /dev/null
\! gpstop -u > /dev/null
select * from diskquota.status();

\! gpconfig -c "diskquota.hard_limit" -v "false" > /dev/null
\! gpstop -u > /dev/null
select * from diskquota.status();

select from diskquota.pause();
select * from diskquota.status();

\! gpconfig -c "diskquota.hard_limit" -v "on" > /dev/null
\! gpstop -u > /dev/null
select * from diskquota.status();

\! gpconfig -c "diskquota.hard_limit" -v "false" > /dev/null
\! gpstop -u > /dev/null
select * from diskquota.status();

select from diskquota.resume();
\! gpconfig -c "diskquota.hard_limit" -v "false" > /dev/null
\! gpstop -u > /dev/null
select * from diskquota.status();
