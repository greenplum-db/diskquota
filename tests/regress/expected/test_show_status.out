select * from diskquota.status() where name not like '%version';
    name     | status 
-------------+--------
 soft limits | on
 hard limits | off
(2 rows)

\! gpconfig -c "diskquota.hard_limit" -v "on" > /dev/null
\! gpstop -u > /dev/null
select * from diskquota.status() where name not like '%version';
    name     | status 
-------------+--------
 soft limits | on
 hard limits | off
(2 rows)

\! gpconfig -c "diskquota.hard_limit" -v "off" > /dev/null
\! gpstop -u > /dev/null
select * from diskquota.status() where name not like '%version';
    name     | status 
-------------+--------
 soft limits | on
 hard limits | off
(2 rows)

select from diskquota.pause();
--
(1 row)

select * from diskquota.status() where name not like '%version';
    name     | status 
-------------+--------
 soft limits | off
 hard limits | off
(2 rows)

\! gpconfig -c "diskquota.hard_limit" -v "on" > /dev/null
\! gpstop -u > /dev/null
select * from diskquota.status() where name not like '%version';
    name     | status 
-------------+--------
 soft limits | off
 hard limits | off
(2 rows)

\! gpconfig -c "diskquota.hard_limit" -v "off" > /dev/null
\! gpstop -u > /dev/null
select * from diskquota.status() where name not like '%version';
    name     | status 
-------------+--------
 soft limits | off
 hard limits | off
(2 rows)

select from diskquota.resume();
--
(1 row)

\! gpconfig -c "diskquota.hard_limit" -v "off" > /dev/null
\! gpstop -u > /dev/null
select * from diskquota.status() where name not like '%version';
    name     | status 
-------------+--------
 soft limits | on
 hard limits | off
(2 rows)

